src/main/java/core/basesyntax/service/FileService.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface FileService {
+    List<String> read(String fileName);
+
+    void writeToFile(String content, String fileName);
```

Comment:
The `writeToFile` method should take a `List<String>` or similar collection as input instead of a single `String` to allow writing multiple lines to a file.